src/setuptools_scm/file_finder_git.py

```diff
@@ -46,7 +47,8 @@ def _git_ls_files_and_dirs(toplevel):
     cmd = ["git", "archive", "--prefix", toplevel + os.path.sep, "HEAD"]
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, cwd=toplevel)
     try:
-        return _git_interpret_archive(proc.stdout, toplevel)
+        with closing(proc.stdout):
```

Comment:
the process is never `.wait()`ed on (iirc `stdout` gets properly cleaned up in that case and isn't triggering the resource warning?) should there be a `finally: proc.wait()`?